sdk/core/core-rest-pipeline/src/interfaces.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/// <reference lib="dom" />
```

Comment:
Can we avoid this by defining an empty version of `ReadableStream` inside `core-rest-pipleine-shims.d.ts`?

Reply:
Agreed. This will pollute every core-rest-pipeline consumer's types with DOM types. It also _shouldn't_ be necessary. DOM types should already be loaded in this package's configuration.

Reply:
Thanks for suggesting the shims. I agree, I'll do that instead